ui.frontend/src/main/webpack/site/js/utils/saasStyle.ts

```diff
@@ -1,6 +1,11 @@
 const initSaasStyle = (): void => {
-  document.styleSheets[0].insertRule(
-    `
+    const style = document.createElement("style");
+    document.head.prepend(style); // must append before we can access sheet property
```

Comment:
method: "prepend"
comment: "append"
which one is it now? 😅

Reply:
fixed